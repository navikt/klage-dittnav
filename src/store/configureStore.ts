import { applyMiddleware, createStore } from 'redux';
import thunkMiddleware from 'redux-thunk';
import reducer from './reducer';

export default function () {
    return createStore(reducer, applyMiddleware(thunkMiddleware));
}
