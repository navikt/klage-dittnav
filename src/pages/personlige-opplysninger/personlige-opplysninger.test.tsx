import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import configureStore from '../../store/configureStore';
import PersonligeOpplysningerPage from './personlige-opplysninger-page';

describe('Welcome klage', () => {
    const store = configureStore();

    it('Renders correctly when there is no data', () => {
        const tree = renderer
            .create(
                <Provider store={store}>
                    <PersonligeOpplysningerPage />
                </Provider>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
