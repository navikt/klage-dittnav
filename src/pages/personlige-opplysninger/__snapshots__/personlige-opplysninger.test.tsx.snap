// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome klage Renders correctly when there is no data 1`] = `
<svg
  aria-label="Laster innhold"
  className="spinner spinner--xl"
  focusable="false"
  height={32}
  kind="spinner-negativ"
  preserveAspectRatio="xMidYMid"
  viewBox="0 0 50 50"
  width={32}
>
  <title>
    Venter...
  </title>
  <circle
    cx="25"
    cy="25"
    fill="none"
    r="20"
    stroke="#eee"
    strokeWidth="5"
    xmlns="http://www.w3.org/2000/svg"
  />
  <circle
    cx="25"
    cy="25"
    fill="none"
    r="20"
    stroke="#888"
    strokeDasharray="50 155"
    strokeLinecap="round"
    strokeWidth="5"
  />
</svg>
`;
